.circleci/config.yml

```diff
@@ -41,17 +47,22 @@ jobs:
           command: npm run test:coverage
       - store_test_results:
           path: test_results
+
   build:
     executor: main-executor
     steps:
-      - checkout_from_cache
+      - checkout_from_workspace
       - run:
           name: build
           command: npm run build
+      - persist_to_workspace:
+          root: .
+          paths: dist
```

Comment:
add the dist directory to the workspace

Reply:
I changed root to `root: ~/webex-js-sdk` (or in your case `root: ~/webex-components`) since I didn't know what `./` was pointing too

Reply:
It looks at workspace definition. . Is fine